workflow-templates/im-test-testng-schedule.yml

```diff
@@ -0,0 +1,173 @@
+﻿# Workflow Code: LazySnorlax_v1   DO NOT REMOVE
+# Purpose:
+#       Runs Java Selenium UI Health Check tests in Github Actions on a schedule.
+#       With the workflow you specify when to run this test.
+#
+# Frequency:
+#    - This workflow can be duplicated once per testng project that you wish to run independently.
+#
+# Projects to use this Template with:
+#    - Testing projects that use Selenium Java with TestNg.
+#
+# TODO Prerequisites:
+#    - Make sure you have the Javaagent and applicationinsights.json in resources folder.
+#    - Update your java selenium solution to export test results to Application Insights using the im.exporttoappinsights jar dependencies. 
+#    - Add secret APPLICATIONINSIGHTS_CONNECTION_STRING to every environment you are testing.
+#    - Modify your Maven profile fetch artifacts from github packages.
+#      Detailed setup guidance can be found here:  https://github.com/im-practices/techhub/blob/main/docs/Azure/content/Health-Checks/TestNG-UI-Health-Check.md
+#    - Create the shell script to setup Maven with the modified settings.xml file.details are at the following link Step 4,5
+#    - https://github.com/im-practices/techhub/blob/main/docs/Azure/content/Health-Checks/TestNG-UI-Health-Check.md
+name: 🧪 Run Java Selenium TestNG tests on Schedule
```

Comment:
Add a line between the comments and the name for better readability

Reply:
added line